pkg/ai/prompts.go

```diff
@@ -8,10 +8,51 @@ const (
 	`
 	trivy_vuln_prompt = "Explain the following trivy scan result and the detail risk or root cause of the CVE ID, then provide a solution. Response in %s: %s"
 	trivy_conf_prompt = "Explain the following trivy scan result and the detail risk or root cause of the security check, then provide a solution."
+
+	prom_conf_prompt = `Simplify the following Prometheus error message delimited by triple dashes written in --- %s --- language; --- %s ---.
+	This error came from the Prometheus configuration file, documented at https://prometheus.io/docs/prometheus/latest/configuration/configuration/.
```

Comment:
I am thinking about the value of having the link in the prompt. The LLM will surely indicate the link in the response, and we might face confusion regarding the Prometheus version (e.g., my current Prometheus version may not be the latest, but the link indicates the latest). What do you think?